modules/xfem/test/tests/solid_mechanics_basic/edge_crack_3d_mhs.i

```diff
@@ -0,0 +1,179 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+  volumetric_locking_correction = true
```

Comment:
I think we do not need this.

Reply:
Most of the input files under the same test folder "solid_mechanics_basic" has the option of `volumetric_locking_correction` turned on, e.g. edge_crack_3d.i and penny_crack.i.  That's where the new input files were derived from.  I left the option on for consistency with other input files.